spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -262,10 +262,22 @@ const ConfigSteps = () => {
 			analyticsClient.sendUIEvent({ actionSubject: "installToNewOrganisation", action: "clicked"}, { mode });
 			await AppManager.installNewApp({
 				onFinish: async (gitHubInstallationId: number | undefined) => {
-					analyticsClient.sendTrackEvent({ actionSubject: "installNewOrgInGithubResponse", action: gitHubInstallationId ? "success" : "fail"}, { mode });
-					getOrganizations();
-					if(gitHubInstallationId) {
-						await doCreateConnection(gitHubInstallationId, "auto");
+					analyticsClient.sendTrackEvent(
+						{
+							actionSubject: "installNewOrgInGithubResponse",
+							action: gitHubInstallationId ? "success" : "fail",
+						},
+						{ mode }
+					);
+					const orgsResults = await getOrganizations();
+					let orgLogin;
+					if (orgsResults?.success) {
+						const orgs = orgsResults?.orgs;
+						const newOrg = orgs.find((org) => org?.id === gitHubInstallationId);
```

Comment:
\nit 
Don't see the need for this extra variable here. Can just do `orgsResults?.orgs.find....`

Reply:
Done